Reply/app/src/main/java/com/materialstudies/reply/util/WindowSizeUtil.kt

```diff
@@ -0,0 +1,14 @@
+package com.materialstudies.reply.util
+
+import androidx.compose.material3.windowsizeclass.WindowSizeClass
+import androidx.compose.material3.windowsizeclass.WindowWidthSizeClass
+
+enum class ReplyContentType {
+    SINGLE_PANE, TWO_PANE
+}
+
+fun getContentType(windowSize: WindowSizeClass) =
```

Comment:
Nit: This can also be an extension function on WindowSizeClass for better scoping.